Structure_Engine/Query/Tangent.cs

```diff
@@ -32,9 +35,13 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Gets the tangent vector of a bar as the direction vector from start to end node. No offsets or similar are taken into account.")]
```

Comment:
"Gets the tangent Vector of a Bar as the direction Vector from StartNode to EndNode. No offsets or similar are taken into account.